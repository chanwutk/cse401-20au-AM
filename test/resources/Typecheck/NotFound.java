class NotFound {
	public static void main(String[] args) {
		System.out.println(new X().f(0));
	}
}

class C1 extends Y {
	public int f(int x) {
		return y;
	}
}

class C2 {
	public int f(int x) {
		return this.g(x);
	}
}
