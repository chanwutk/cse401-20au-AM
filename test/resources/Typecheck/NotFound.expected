NotFound.java:7: error: cannot find symbol
  symbol:   class Y
NotFound.java:3: error: cannot find symbol
  symbol:   class X
  location: class NotFound
NotFound.java:9: error: cannot find symbol
  symbol:   variable y
  location: class C1
NotFound.java:15: error: cannot find symbol
  symbol:   method g(...)
